angular/src/app/models/parkeerkans.ts

```diff
@@ -0,0 +1,7 @@
+export interface Parkeerkans {
+  [key: string]: Wegdeel;
+}
+
+interface Wegdeel {
```

Comment:
Dit is specifiek voor hoe informatie over een wegdeel terug komt in de parkeerkans response